Code_of_Conduct.md

```diff
@@ -0,0 +1,65 @@
+
+Presenters are asked to avoid language which is not appropriate for an all-ages audience as much as possible. Please use English wherever possible.
+
+If the subject matter of the presentation cannot be presented adequately without including language that could be considered offensive, this should be pointed out in advance, at the beginning of the talk and in the schedule. Failing to do so, will lead to actions mentioned in ["What does that mean for me?"](#What-does-that-mean-for-me?) section.
+
+If presenters are unsure whether their material is suitable, they are encouraged to show it to the event’s speaker liaison or send the same for reviews to admins before their session.
+
+## Be aware.
+
+
+We ask everyone to be aware that we will not tolerate intimidation, harassment, or any abusive, discriminatory or derogatory behaviour by anyone at any event or online.
+
+Complaints can be made to the admins or teacher representative of the group meeting them in-person contacting them through direct messages. All complaints made to event admins will remain confidential and be taken seriously. The complaint will be treated appropriately with discretion. Should admins or teacher representative consider it appropriate, measures they may take include:
+
+    - the individual(s) may be told to apologise
+    - the individual(s) may be told to stop/modify their behaviour appropriately
+    - the individual(s) may be warned that enforcement action may be taken if the behaviour continues
+    - the individual(s) may be asked to immediately leave the venue and/or will be prohibited from continuing to attend the event
+    - the incident may be reported to the appropriate authorities
+    - the admins/organisers can ban the individual(s) for unforeseeable future
+
+## What does that mean for me?
+
+
+All participants, including event attendees and speakers must not engage in any intimidation, harassment, or abusive or discriminatory behaviour.
+
+Here are some examples of behaviours that have occurred at past events which are not appropriate:
+
+    - offensive verbal or written remarks related to gender, sexual orientation, disability, physical appearance, body size, race or religion;
+    - sexual or violent images in public spaces (including presentation slides);
+    - deliberate intimidation;
+    - stalking or following;
+    - unwanted photography or recording;
+    - sustained disruption of talks or other events;
+    - intoxication at an event venue;
+    - inappropriate physical contact;
+    - unwelcome sexual attention;
+    - sexist, racist, or other exclusionary jokes;
+    - unwarranted exclusion from conference or related events based on age, gender, sexual orientation, disability, physical appearance, body size, race, religion;
+
+We want everyone to have a good time at our events. Be aware, be inclusive, be polite, be respectful and if you observe anything violating CoC, please report immediately to any admins of the group.
```

Comment:
I think it is in the best practice to provide an email ID and maybe a number, too.
Also, you need to specify which `group` you're talking about. Not everyone who's a victim of violation is a member of the `group` you're referring to.